### DISCUSSION TITLE
Cannot enable Imei changer pro xposed module

### DISCUSSION THREAD

**Discussion by arismfg** [LSPosed/LSPosed#477] [Q&A] [answered]
Hi, I am trying to use xposed imei changer pro. Lsposed can see the module grayed out but when I try to enable the module, it cannot find the imei changer app in installed application list to select it. So, I cannot enable the module even though it knows it is there. Imei changer pro app is definately installed and can see in phone apps
Using riru 25.3.4 and magisk 20.4. 

Log does not give any log for that.

Any help appreciated. I am new to lsposed, so not sure if doing something wrong

**Comment by yujincheng08** [comment]
modules' scopes always include themselves.

**Comment by arismfg** [comment]
Can you explain more what you mean? I cannot find the application name to select it and enable the module

**Comment by yujincheng08** [comment]
I mean you don't have to select imei changer itself in its scope list.

**Comment by arismfg** [comment]
Ok, so I can select any app?

**Comment by yujincheng08** [comment, marked as answer]
Select the app that you need to change IMEI for.

But this module is pretty stupid. A well-designed module should only need to select system framework but this one you need to select needed apps.

**Comment by arismfg** [comment]
Do you know any other that can properly change imei using Xposed or not? It is an old module anyway.

And what's the needed apps for this module?
